tasks/Task.cpp

```diff
@@ -26,20 +26,22 @@ Task::~Task()
 
 bool Task::configureHook()
 {
-    if (! TaskBase::configureHook())
-        return false;
+    if (!_io_port.get().empty())
+        lamps.openURI(_io_port.value());
+    else
+        throw std::runtime_error("io_port property not set");
```

Comment:
Relevant checks about io_port is done within the base task. Just openURI if it is set.